clang/test/Driver/ignored-pch.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+
+// Create PCH without -ignore-pch.
+// RUN: %clang -x c++-header %S/../Modules/Inputs/codegen-flags/foo.h -### 2>&1 | FileCheck %s -check-prefix=CHECK-EMIT-PCH
```

Comment:
`%S/../` references auxiliary file from a different test directory. We typically avoid doing this,,, though Index/ and Headers/ seem to have violations.

Reply:
Hi @MaskRay,

Could you please let me know whether you are happy with the change?

Thanks,

Reply:
Hi @MaskRay,

Assuming you are happy with the change. I will commit the change later today if I haven't received any further comments.

Thanks,